ui/src/i18n/de.json

```diff
@@ -232,11 +232,11 @@
     "restart.toast_server_restart_error": "Beim Senden des Neustartbefehls an den Server ist ein Fehler aufgetreten",
     "restart.toast_server_restarted": "Server wurde neu gestartet",
     "restart.toast_sever_restart_timeout": "Der Server braucht lange, um wieder online zu gehen",
-    "settings.manage_cache.help": "Wenn ein nicht mehr vorhandenes Gerät noch in HomeKit angezeigt wird, müssen Sie es möglicherweise aus dem Cache entfernen. Sie können entweder ein einzelnes Gerät aus dem Cache entfernen oder alle zwischengespeicherten Geräte entfernen.",
-    "settings.reset.help": "Wenn Sie Probleme beim Koppeln von Homebridge oder einem externem Gerät wie einer Kamera oder einem Fernseher mit HomeKit haben, müssen Sie möglicherweise das Gerät entkoppeln oder Ihre Homebridge-Instanz zurücksetzen.",
+    "settings.manage_cache.help": "Wenn ein nicht mehr vorhandenes Gerät noch in HomeKit angezeigt wird, musst du es möglicherweise aus dem Cache entfernen. Du kannst entweder ein einzelnes Gerät oder alle zwischengespeicherten Geräte aus dem Cache entfernen.",
+    "settings.reset.help": "Wenn du Probleme beim Koppeln von Homebridge oder einem externem Gerät wie einer Kamera oder einem Fernseher mit HomeKit hast, musst du möglicherweise das Gerät entkoppeln oder deine Homebridge-Instanz zurücksetzen.",
     "settings.title_actions": "Aktionen",
     "settings.title_environment_variables": "Umgebungsvariablen",
-    "settings.title_manage_cached_accessories": "Verwalten von gecachten Geräten",
+    "settings.title_manage_cached_accessories": "Verwalten von Geräten im Cache",
```

Comment:
In my opinion this translation isn't better than the one before.
I would not translate it this way.